20.Mender-client/docs.md

```diff
@@ -7,6 +7,152 @@ shortcode-core:
 github: false
 ---
 
+## mender 4.0.0
+
+_Released 01.15.2024_
+
+### Statistics
+
+A total of 70335 lines added, 61546 removed (delta 8789)
+
+| Developers with the most changesets | |
+|---|---|
+| Kristian Amlie | 478 (47.5%) |
+| Ole Petter Orhagen | 246 (24.4%) |
+| Vratislav Podzimek | 155 (15.4%) |
+| Lluis Campos | 104 (10.3%) |
+| Peter Grzybowski | 4 (0.4%) |
+| Lars Erik Wik | 4 (0.4%) |
+| Lukasz Finster | 3 (0.3%) |
+| Adam Duskett | 2 (0.2%) |
+| Alan | 2 (0.2%) |
+| Alf-Rune Siqveland | 2 (0.2%) |
+
+| Developers with the most changed lines | |
+|---|---|
+| Kristian Amlie | 78431 (66.4%) |
+| Ole Petter Orhagen | 15419 (13.1%) |
+| Vratislav Podzimek | 14779 (12.5%) |
+| Lluis Campos | 7198 (6.1%) |
+| Peter Grzybowski | 1055 (0.9%) |
+| Adam Duskett | 535 (0.5%) |
+| Lars Erik Wik | 330 (0.3%) |
+| Lukasz Finster | 138 (0.1%) |
+| Josef Holzmayr | 114 (0.1%) |
+| Alf-Rune Siqveland | 42 (0.0%) |
+
+| Developers with the most lines removed | |
+|---|---|
+| Kristian Amlie | 19656 (31.9%) |
+| Adam Duskett | 132 (0.2%) |
+| Josef Holzmayr | 81 (0.1%) |
+| Fabio Tranchitella | 1 (0.0%) |
+
+| Developers with the most signoffs (total 6) | |
+|---|---|
+| Vratislav Podzimek | 3 (50.0%) |
+| Ole Petter Orhagen | 2 (33.3%) |
+| Kristian Amlie | 1 (16.7%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 1001 (99.4%) |
+| Tronel | 3 (0.3%) |
+| aduskett@gmail.com | 2 (0.2%) |
+| BlackBerry Limited | 1 (0.1%) |
+
+| Top lines changed by employer | |
+|---|---|
+| Northern.tech | 117379 (99.4%) |
+| aduskett@gmail.com | 535 (0.5%) |
+| Tronel | 138 (0.1%) |
+| BlackBerry Limited | 32 (0.0%) |
+
+| Employers with the most signoffs (total 6) | |
+|---|---|
+| Northern.tech | 6 (100.0%) |
+
+| Employers with the most hackers (total 16) | |
+|---|---|
+| Northern.tech | 13 (81.2%) |
+| aduskett@gmail.com | 1 (6.2%) |
+| Tronel | 1 (6.2%) |
+| BlackBerry Limited | 1 (6.2%) |
+
+### Changelogs
+
+#### mender (4.0.0)
+
+New changes in mender since 3.5.2:
```

Comment:
No mentions this is the "big rewrite project"?

Reply:
I can add here some note pointing to the mender-docs page that explains the upgrade and split. It depends on:
* https://github.com/mendersoftware/mender-docs/pull/2289

Reply:
I can add here some note pointing to the mender-docs page that explains the upgrade and split. It depends on:
* https://github.com/mendersoftware/mender-docs/pull/2289

Reply:
Ah yes, I was planning to make a bigger note about it. If you want, I can take over this PR, and combine with my work in the other PR. Just assign it to me.

Reply:
Thank you :pray:

Reply:
Mentioned the rewrite at the top level, with a link to the documentation for the migration guide.